cmd/cloud/installation.go

```diff
@@ -51,13 +51,14 @@ func init() {
 
 	installationListCmd.Flags().String("owner", "", "The owner by which to filter installations.")
 	installationListCmd.Flags().String("group", "", "The group ID by which to filter installations.")
+	installationListCmd.Flags().String("state", "", "The state by which to filter results by.")
+	installationListCmd.Flags().String("dns", "", "The dns by which to filter results by.")
```

Comment:
0/5 nitpick grammar time:

"The dns by which to filter results" -- the second 'by' is redundant since you used the 'by which to' form.

Reply:
Yeah, I kinda thought the same, but just wanted it to be similar in wording to the other flags for now. I think I will go back and update them all on a new PR I have underway.

Thanks for the input!